### PR TITLE
Adds support for overriding imagepullsecrets and image registry globally

### LINKED ISSUE #47: Support global custom image registries for private clusters on the helm chart
**Is your feature request related to a problem? Please describe.**
The chart does note support using a global override for image registries

**Describe the solution you'd like**
Add conditions on the template for global custom image registries and image pull secrets matching other charts.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by ptcar2009**
closes #47

### CODE REVIEW

File: deploy/helm-chart/kubernetes-secret-generator/templates/_helpers.tpl [hunk 1]
```diff
@@ -72,3 +72,40 @@ Define the namespace to watch
     {{ .Values.watchNamespace }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Return the proper Docker Image Registry Secret Names
+{{ include "common.images.pullSecrets" ( dict "images" (list .Values.path.to.the.image1, .Values.path.to.the.image2) "global" .Values.global) }}
```
Comment (hensur) [inline deploy/helm-chart/kubernetes-secret-generator/templates/_helpers.tpl:78]:
```suggestion
{{ include "common.images.pullSecrets" ( dict "images" (list .Values.image) "global" .Values.global) }}
```

Comment (hensur) [reply deploy/helm-chart/kubernetes-secret-generator/templates/_helpers.tpl:78]: It's just a comment, but I think it looks cleaner this way :)

**Review by hensur — CHANGES_REQUESTED** [review]

File: deploy/helm-chart/kubernetes-secret-generator/templates/_helpers.tpl (hunk 1 above)
Comment (ptcar2009) [reply deploy/helm-chart/kubernetes-secret-generator/templates/_helpers.tpl]: I agree, I got the helpers comment from bitnami's charts

**Comment by ptcar2009** [comment]
@hensur can you take a look at it again? it's a pretty simple change and shouldn't really affect anything

**Review by martin-helmich — APPROVED** [review]
Sorry for the delay. 🙄🙏  Thanks for your contribution! :+1:

**Comment by ptcar2009** [comment]
@martin-helmich sorry for bothering again, but is there any way to create a new version of the helm chart containing this change?

**Comment by day1118** [comment]
> @martin-helmich sorry for bothering again, but is there any way to create a new version of the helm chart containing this change?

Can you please cut a release @martin-helmich @hensur?
Thanks

**Pull request merged**